caluma/caluma_form/validators.py

```diff
@@ -184,7 +184,7 @@ def validate(self, *, question, document, info, validation_context=None, **kwarg
 
 
 class DocumentValidator:
-    def validate(self, document, info, validation_context=None, **kwargs):
+    def validate(self, document, user, validation_context=None, **kwargs):
```

Comment:
I love that you refactored that dreaded `info` out of the validator! I wanted to get rid of it for a while now :)